internal/pipeline.go

```diff
@@ -52,7 +52,7 @@ type NewComponent func() Component
 var registeredComponents map[string]NewComponent
 
 // NewPipeline : Using a list of strings holding name of components, create and configure the component objects
-func NewPipeline(components []string) (*Pipeline, error) {
+func NewPipeline(components []string, validateCredentials bool) (*Pipeline, error) {
```

Comment:
same here, no validate-creds just isDaemon or something like that.

Reply:
renamed to isParent